Italian/bagno/doccia/stefy/stefy.md

```diff
@@ -0,0 +1,5 @@
+..e tu sei mio prigioniero.
+
+Siamo in una bolla di sapone. Isolati dal mondo.
+
+Puoi decidere di fare [ciò che dico io](dico-io/dico-io.md) oppure [no](no/no.md). A te la scelta. 
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.